cmd/arcaflow/main.go

```diff
@@ -51,61 +53,50 @@ func main() {
 		Stdout:      os.Stderr,
 	})
 
+	defaultConfig := `
+log:
+  level: info
+deployers:
+  image:
+    deployer_name: podman
+    deployment:
+      imagePullPolicy: IfNotPresent
+logged_outputs:
+  error:
+    level: info`
+
 	configFile := ""
 	input := ""
 	dir := "."
 	workflowFile := "workflow.yaml"
 	printVersion := false
 
-	flag.BoolVar(&printVersion, "version", printVersion, "Print Arcaflow Engine version and exit.")
-	flag.StringVar(
-		&configFile,
-		"config",
-		configFile,
-		"The Arcaflow configuration file to load, if any.",
+	const (
+		helpUsage     = "Display this help output."
+		versionUsage  = "Print Arcaflow Engine version and exit."
+		configUsage   = "The path to the Arcaflow configuration file to load, if any."
+		inputUsage    = "The path to the workflow input file to load."
+		contextUsage  = "The path to the workflow directory to run from."
+		workflowUsage = "The path to the workflow file to load."
 	)
-	flag.StringVar(
-		&input,
-		"input",
-		input,
-		"The workflow input file to load. May be outside the workflow directory. If no input file is passed, "+
-			"the workflow is assumed to take no input.",
-	)
-	flag.StringVar(
-		&dir,
-		"context",
-		dir,
-		"The workflow directory to run from. Defaults to the current directory.",
-	)
-	flag.StringVar(
-		&workflowFile,
-		"workflow",
-		workflowFile,
-		"The workflow file in the current directory to load. Defaults to workflow.yaml.",
-	)
-	flag.Usage = func() {
-		_, _ = os.Stderr.Write([]byte(`Usage: arcaflow [OPTIONS]
-
-The Arcaflow engine will read the current directory and use it as a context
-for executing the workflow.
-
-Options:
+	flag.BoolVar(&printVersion, "version", printVersion, versionUsage)
+	flag.StringVar(&configFile, "config", configFile, configUsage)
+	flag.StringVar(&input, "input", input, inputUsage)
+	flag.StringVar(&dir, "context", dir, contextUsage)
+	flag.StringVar(&workflowFile, "workflow", workflowFile, workflowUsage)
 
-  -version            Print the Arcaflow Engine version and exit.
-
-  -config FILENAME    The Arcaflow configuration file to load, if any.
-
-  -input FILENAME     The workflow input file to load. May be outside the
-                      workflow directory. If no input file is passed,
-                      the workflow is assumed to take no input.
-
-  -context DIRECTORY  The workflow directory to run from. Defaults to the
-                      current directory.
-
-  -workflow FILENAME  The workflow file in the current directory to load.
-                      Defaults to workflow.yaml.
-`))
+	flag.Usage = func() {
+		w := flag.CommandLine.Output()
+		_, _ = w.Write(
+			[]byte(
+				"Usage: " + os.Args[0] + " [OPTIONS]\n\n" +
+					"The Arcaflow engine will read the current directory and use it as a " +
+					"context for executing the workflow.\n\n",
```

Comment:
Yeah: the documentation is inconsistent with the implementation, so one or both should change. And if we always resolve paths relative to the working directory, I don't see how `--context` has any reason to exist.

Reply:
I believe that the code as it presently stands resolves relative paths relative to the context directory which defaults to the current working directory...so that bit of documentation is accurate in the absence of using `--context`.  😇

Reply:
I think it's best just to remove this extra help output as it is inaccurate and confusing.

Reply:
I actually ended up updating this instead of removing in due to other comments on the PR.